libraries/chain/asset_evaluator.cpp

```diff
@@ -447,8 +458,12 @@ static void update_bitasset_object_options(const asset_update_bitasset_operation
       }
    }
 
+   const auto old_feed_price = bdo.current_feed.settlement_price;
    if( should_update_feeds )
       bdo.update_median_feeds(db.head_block_time());
+
+   if (after_hardfork_890 && old_feed_price != bdo.current_feed.settlement_price)
+      db.check_call_orders(asset_to_update);
```

Comment:
1. This should only be done in `if( should_update_feeds )`.
2. Although not very sure, I think it's better to call `check_call_orders` out of the `modify` lambda. Perhaps let this static function return a `bool` and call `check_call_orders` accordingly.

Reply:
Done, at the expense of adding some things to the capture list. Please verify my understanding of the logic:
IF (after hardfork 890) AND (feed price changed) THEN (check_call_orders)

Reply:
> IF (after hardfork 890) AND (feed price changed) THEN (check_call_orders)

Correct.